osu.Game/Screens/Backgrounds/BackgroundScreenBeatmap.cs

```diff
@@ -60,6 +60,19 @@ public BackgroundScreenBeatmap(WorkingBeatmap beatmap = null)
             StoryboardReplacesBackground.BindTo(dimmable.StoryboardReplacesBackground);
         }
 
+        public void TurnFitMode(bool On)
+        {
+            cancellationSource?.Cancel();
+            if (On)
+            {
+                LoadComponentAsync(BeatmapBackground.FittedBeatmapBackground(beatmap), switchBackground, (cancellationSource = new CancellationTokenSource()).Token);
```

Comment:
Reloading the entire background seemingly just to change its sprite's fill mode seems excessive.